Modules/CodeTrek/Config/config.php

```diff
@@ -17,6 +17,11 @@
         ],
     ],
     'rounds' =>[
+        'introductory-call' => [
+            'label' => 'Introductory Call',
+            'slug'  => 'introductory-call',
+            'class' => 'badge badge-warning'
```

Comment:
I think the `badge-warning` color does not make sense in the `introductory-call` round. I suggest make the custom class. and then used directly here.
Something like this.
```
.badge-theme-teal {
background-color: #63b2b7;
}
```
```suggestion
            'class' => 'badge badge-theme-teal'
```